dask/dataframe/rolling.py

```diff
@@ -110,10 +113,15 @@ def _call_method(self, method_name, *args, **kwargs):
             self.rolling_kwargs, method_name, args, kwargs)}
         if self.rolling_kwargs['axis'] in [0, 'rows']:
             # roll in the partition direction (will need to access neighbor)
+            window = self.rolling_kwargs['window']
+            tail_name = '{}-tailprev-{}'.format(old_name, window-1)
```

Comment:
I wasn't sure what naming convention to use given your example. I chose 'tailprev' for the descriptive part because it captures what I'm doing with some subtlety while still being brief. I put the window size in directly rather than tokenizing, since it seems silly to make it less accessible. I assume `old_name` already gives us the uniqueness we need?

Thanks again for your help and guidance.